mlflow/tracking/fluent.py

```diff
@@ -25,7 +25,7 @@
 
 _EXPERIMENT_ID_ENV_VAR = "MLFLOW_EXPERIMENT_ID"
 _RUN_ID_ENV_VAR = "MLFLOW_RUN_ID"
-_active_run = None
+_active_run_stack = []
```

Comment:
I guess we already didn't do this, but could be worth documenting somewhere that methods in the fluent API aren't thread-safe (since they operate on a process-wide global active run)